index.html

```diff
@@ -232,40 +229,16 @@ <h3 class="text-colored text-center">Giga</h3>
 				</div>
 			</div>
 
-			<!--
-			<div class="row">
-				<div class="col-md-12 top50">
-					<h3 class="text-colored text-center">Mega</h3>
-				</div>
-			</div>
-
-			<div class="row">
-				<div class="col-md-12 top50">
-					<h3 class="text-colored text-center">Kilo</h3>
-				</div>
-			</div>
-			-->
-
-			<div class="row">
-				<div class="col-md-12 top50">
-					<h3 class="text-colored text-center">Scientific Partner</h3>
-				</div>
-			</div>
 			<div class="row sponsors-kilo">
```

Comment:
huh `sponsors-kilo`?

Reply:
Yup because `sponsors-mega` is currently as big as `sponsors-giga`. I think you tuned down the size of `sponsors-giga` a notch some time ago.

Reply:
💩 okay alright for now I guess, will have to deal with that later